src/index.html

```diff
@@ -458,6 +459,11 @@ <h3 class="gallery__heading">Łukasz Nowacki</h3>
     </p>
   </div>
 </footer>
+<!-- Inline for faster parsing -->
+<script>
+  document.querySelector('body').classList.toggle('with-js');
+  document.querySelector('.main-header').classList.toggle('main-header--sticky');
```

Comment:
Done with both comments. Doing the change to document.body wasn't a big deal, so why not do it if it's rejected on CR anyways? ;)